app/components/node/PoSSize.tsx

```diff
@@ -142,6 +152,19 @@ const PoSSize = ({
     setNumUnit(commitments[index].numUnits);
   };
 
+  const handleMaxFileSize = ({ value: string }) => {
+    const val = covertGBToBytes(parseInt(string, 10));
+    if (
+      !val ||
+      val < DEFAULT_POS_MAX_FILE_SIZE || // 2GB
+      val > DEFAULT_POS_MAX_FILE_SIZE * 50 // 100GB
+    ) {
+      return;
```

Comment:
I've found input behavior inconvenient from UX perspective.
Let's make it possible to clear the input (to write number from scratch) and round big numbers to the maximum value.

To make it easier (than introducing validation, disabling submit button and etc) you can attach handlers to `onBlur` event.
But probably some message (in the case of rounding) will be useful.